lighthouse-core/lib/emulation.js

```diff
@@ -41,10 +41,13 @@ const NEXUS5X_USERAGENT = {
     '(KHTML, like Gecko) Chrome/59.0.3033.0 Mobile Safari/537.36'
 };
 
+const LATENCY_FACTOR = 3.75;
+const THROUGHPUT_FACTOR = 1.1;
```

Comment:
nit: should we model both of these as multipliers so it's just multiply for both?